src/app/components/SearchSources.js

```diff
@@ -9,31 +9,30 @@ export default class SearchSources extends React.Component {
 
   updateSearch(event) {
     if (!/\d/.test(event.target.value)) {
-      this.setState({search: event.target.value.substr(0, 10)});
+      this.setState({ search: event.target.value.substr(0, 10) });
     }
   }
 
   render() {
-    let filteredSources = this.props.sources.filter((source) => {
-      return source.name.toLowerCase().indexOf(this.state.search.toLowerCase()) !== -1;  
-    });
-    
-    let filteredList = filteredSources.map((source) => {
-      return <Source key={source.id} source={source} headlines={this.props.headlines} />;
-    });
+    const filteredSources = this.props.sources.filter(source =>
+    source.name.toLowerCase().indexOf(this.state.search.toLowerCase()) !== -1);
+
+    const filteredList = filteredSources.map(source =>
+    <Source key={source.id} source={source} headlines={this.props.headlines} />);
     return (
       <div>
-        <form class="navbar-form navbar-right"> 
+        <form class="navbar-form navbar-right">
```

Comment:
className instead of class like I noted above. You can note this at every other place it is repeated.